examples/rmg/catalysis/ch4_o2/simulate_ch4o2cat_RMS.ipynb

```diff
@@ -0,0 +1,508 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "b1f80c5e",
+   "metadata": {},
+   "source": [
+    "# Import packages\n",
+    "Note: This notebook can take a while (~5 min) for the first time due to compilation"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "82bfbe9d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from pyrms import rms\n",
+    "from diffeqpy import de\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "from julia import Main\n",
+    "%matplotlib inline"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c1fa0586",
+   "metadata": {},
+   "source": [
+    "Note: You will learn more about how to RMG-cat and generate catalyic mechanism in tomorrow's lecture \"RMG-Cat Systems and Surface Reactions\". You can also remove the `#` in the following cell and run it. This can take quite a while. This is an example for docker users."
```

Comment:
Can we make this comment more general? Future users may not know the background that this notebook was created for the 2023 Workshop. Something like "This notebook was originally created as a practice example for 2023 RMG workshop. In order to follow the notebook, you need to first run RMG-cat and generate the example mechanism under `examples/rmg/catalysis/ch4_o2/input.py`. As a shortcut, you can generate the mechanism by removing the `#` in the following cell and running the cell; this may take quite a while. You may need to modify the path to make it consistent with your RMG installation path if you are not using RMG with Docker. For more details about the RMG-cat mechanisms, you can find links to the RMG workshop lectures at https://rmg.mit.edu/resources."